### PR TITLE
🐛 Fixes listing in billing center 🚨

### PR DESCRIPTION
**Pull request by ignapas**
## What do these changes do?
Bring payment methods back
image

Bonus: The payment gateway window is now slightly wider

### Backend
- environs payment-gateway for development
- strip secrets from logs
- refactoring fakes

## How to test
Go to *Billing center > Payment methods* and check the list

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by GitHK — APPROVED** [review]

**Review by pcrespov — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js [hunk 1]
```diff
@@ -101,7 +101,7 @@ qx.Class.define("osparc.desktop.paymentMethods.PaymentMethods", {
         };
         osparc.data.Resources.fetch("paymentMethods", "init", params)
           .then(data => {
-            this.__popUpPaymentGateway(data["paymentMethodId"], data["paymentMethodFormUrl"]);
+            this.__popUpPaymentGateway(data.paymentMethodId, data.paymentMethodFormUrl);
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js:104]: How is it possible that this changed? Is this an indication that the API changed or was is parsed differently?

**Review by pcrespov — COMMENTED** [review]
Do not forget to check the ACK via web-sockets and close the poup window
thx so much!

File: services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js [hunk 2]
```diff
@@ -184,7 +184,7 @@ qx.Class.define("osparc.desktop.paymentMethods.PaymentMethods", {
         spacing: 3,
         height: 150,
```
Comment (jsaq007) [inline services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js:185]: Perhaps we could increase the height of the table. I don't know how many items we might get in here but I have 5 and I need to scroll. What do you think?

**Review by jsaq007 — APPROVED** [review]

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js (hunk 2 above)
Comment (ignapas) [reply services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js:191]: Yes, I tried to quickly fix it but I could not, so I moved on. This is not trying to enhance anything but rather quickly fix some critical bugs

File: packages/service-library/src/servicelib/fastapi/http_client.py [hunk 3]
```diff
@@ -114,8 +114,16 @@ def to_curl_command(request: httpx.Request, *, use_short_options: bool = True) -
 
     # https://curl.se/docs/manpage.html#-H
     # H, --header <header/@file> Pass custom header(s) to server
+
     headers_option = ""
-    if headers := [f'"{k}: {v}"' for k, v in request.headers.items()]:
+    headers = []
+    for key, value in request.headers.items():
+        if "secret" in key.lower() or "pass" in key.lower():
+            headers.append(f'"{key}: *****"')
```
Comment (sanderegg) [inline packages/service-library/src/servicelib/fastapi/http_client.py:122]: sweet

**Review by YuryHrytsuk — APPROVED** [review]

**Comment by ignapas** [comment]
> full stack 💯

hmmm yes sure 😛

**Pull request merged**